src/packages/solid/components/Badge/Badge.types.ts

```diff
@@ -0,0 +1,27 @@
+import type { NodeStyles, TextProps } from '@lightningjs/solid';
+import type { ComponentStyleConfig, NodeStyleSet, Tone, TextStyleSet } from 'types/types.js';
+import type { UIComponentProps } from 'types/interfaces.js';
+
+export interface BadgeProps extends UIComponentProps, BadgeStyleProperties {
+  /**
+   * Badge text
+   */
+  title?: TextProps['children'];
+}
+
+export type BadgeStyleProperties = Partial<{
+  backgroundColor: NodeStyles['color'];
+  textColor: NodeStyles['color'];
+  iconColor: NodeStyles['color'];
+  strokeColor: NodeStyles['color']; // TODO do we use this?
```

Comment:
can you take a look at the web work and see what this is used for, and if we can implement it?